.github/spellcheck-settings.yml

```diff
@@ -0,0 +1,28 @@
+matrix:
+- name: Markdown
+  expect_match: false
+  apsell:
+    lang: en
+    d: en_US
+    ignore-case: true
+  dictionary:
+    wordlists:
+    - .github/wordlist.txt
```

Comment:
Yes. Basically anything failing to pass spell check because it's specific (say Canadian spelling), or a term that isn't "English" but is common parlance in our world, belongs there.